src/index.html

```diff
@@ -59,33 +59,360 @@
     </header>
 
     <main class="catalog">
+
       <div class="catalog__card">
-        Product card
+
+        <img 
+            src="./images/imac.jpeg" 
+            alt="Samsung Galaxy S22 2022 8/128GB Green" 
+            class="catalog__img"
+        >
+        <h1 class="catalog_describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+    
+        <p class="catalog_product-code">
+          Product code: 195434
+        </p>
+    
+        <div>
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <p class="stars__text"> Reviews:&nbsp5</p>
+          </div>
+        </div>
+    
+        <div class="catalog_price">
+          <p class="catalog_price__text"> 
+            Price:
+          </p>
+    
+          <p class="catalog_price__number">
+            &#36;2,199
+          </p>
+        </div>
+        
+        <a href="#button">
+          <button 
+            class="catalog_button" data-qa="card-hover">
+            Buy
+          </button>
+        </a>
       </div>
 
       <div class="catalog__card">
-        Product card
+
+        <img 
+            src="./images/imac.jpeg" 
+            alt="Samsung Galaxy S22 2022 8/128GB Green" 
+            class="catalog__img"
+        >
+        <h1 class="catalog_describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+    
+        <p class="catalog_product-code">
+          Product code: 195434
+        </p>
+    
+        <div 
+          class="stars stars--4" 
+        >
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <p class="stars__text"> Reviews:&nbsp5</p>
+        </div>
+    
+        <div class="catalog_price">
+          <p class="catalog_price__text"> 
+            Price:
+          </p>
+    
+          <p class="catalog_price__number">
+            &#36;2,199
+          </p>
+        </div>
+        
+        <a href="#button">
+          <button 
+            class="catalog_button" data-qa="card-hover">
```

Comment:
Please fix everywhere
```suggestion
          <button 
            class="catalog_button" data-qa="card-hover"
          >
```